addons/policies/bootstrap_entity_policies.json

```diff
@@ -2710,6 +2710,7 @@
                     "entity-type:Process",
                     "entity-type:Connection",
                     "entity-type:ProcessExecution",
+                    "entity-type:File",
```

Comment:
Are you sure we should deny CRUD explicitly $guest role?

Reply:
https://atlanhq.slack.com/archives/C050G28HZU1/p1684409209818709?thread_ts=1684266830.152639&cid=C050G28HZU1